src/shared/icons/index.ts

```diff
@@ -49,13 +55,19 @@ const icons = [
   'chevron-left',
   'check',
   'dash',
+  'kebab-menu',
+  'pencil',
+  'pencil2',
```

Comment:
Maybe let's try to make the name describe the difference? It will be easy to get lost in what `pencil` vs `pencil2` is. Maybe something like `big-pencil` and `small-pencil` 😆 At least it will give us _some_ info I guess :D

Reply:
haha will try to figure out something 👌